config-service-impl/src/test/java/org/hypertrace/config/service/store/DocumentConfigStoreTest.java

```diff
@@ -402,47 +383,39 @@ void buildQuery_withDefaultPagination() throws Exception {
             SortingSpec.of(
                 IdentifierExpression.of("creationTimestamp"),
                 org.hypertrace.core.documentstore.expression.operators.SortOrder.DESC));
-    assertEquals(expectedSorts, capturedQuery.getSorts());
+
+    configStore.getAllConfigs(
+        new ConfigResource(RESOURCE_NAMESPACE, RESOURCE_NAME, "tenant1"),
+        Filter.getDefaultInstance(),
+        Pagination.getDefaultInstance(),
+        emptyList());
+
+    verify(collection)
```

Comment:
This is what I was suggesting in the other PR - arg captors aren't needed, just call the public API and verify that the query looks like you want it to.